HashRegistrarSimplified.sol

```diff
@@ -237,6 +238,27 @@ contract Registrar {
         }
         return len;
     }
+    
+    /** 
+     * @dev Creates a scheduled release for each name
+     * 
+     * Each name will be assigned a random date in which its auction 
+     * can be started, from 0 to 6 months
+     * 
+     * @param _hash The hash to start an auction on
+     * @param _timestamp The timestamp to query about
+     */
+     
+    function isAllowed(bytes32 _hash, uint _timestamp) constant returns (bool allowed){
```

Comment:
The problem doing so is that it should be a rather simple calculation on the client side: `registryStarted + launchLength * myHash/0xFFF`, but it runs on two issues doing on the blockchain using uints: you can't do (launchLength * myHash) because it might go over the overflow limit, and you can't invert it and do (0xFFF / myHash) because it returns a non-integer.